packages/mini-browser/src/browser/mini-browser-content-style.ts

```diff
@@ -0,0 +1,32 @@
+/********************************************************************************
+ * Copyright (C) 2019 Red Hat, Inc. and others.
```

Comment:
`TypeFox`, since you've extracted it from mini-browser-content.ts

Reply:
done